USAGE_desktop.md

```diff
@@ -119,66 +123,12 @@ export VK_INSTANCE_LAYERS=VK_LAYER_LUNARG_gfxreconstruct
 ### Capture Options
 
 The GFXReconstruct layer supports several options, which may be enabled
-through environment variables or a layer settings file.
-
-#### Windows Options
-
-The following example demonstrates how to set the layer's log level to
-"warning" from the Windows command prompt:
-
-```bat
-set GFXRECON_LOG_LEVEL=warning
-```
-
-#### Linux Options
-
-The following example demonstrates how to set the layer's log level to
-"warning" from the Linux command line:
-
-```bash
-export GFXRECON_LOG_LEVEL=warning
-```
-
-##### Supported Options
-
-Options with the BOOL type accept the following values:
-
-* A case-insensitive string value 'true' or a non-zero integer value indicate
-  true.
-* A case-insensitive string value 'false' or a zero integer value indicate
-  false.
-
-The capture layer will generate a warning message for unrecognized or invalid
-option values.
-
-Option | Environment Variable | Type | Description
-------| ------------- |------|-------------
-Capture File Name | GFXRECON_CAPTURE_FILE | STRING | Path to use when creating the capture file.  Default is: `gfxrecon_capture.gfxr`
```

Comment:
Many of these options as environment variables seem to have been lost in this PR.  I just tried searching a few - like `GFXRECON_CAPTURE_FILE` is mentioned in the new `USAGE_desktop.md` but GFXRECON_CAPTURE_COMPRESSION_TYPE, GFXRECON_CAPTURE_TRIGGER, and GFXRECON_LOG_BREAK_ON_ERROR are missing in a random sampling.

Reply:
Those options are now documented in auto-generated documentation files. You can view the auto-generated html gfxr document in Vulkan-SDK-Packaging/Guides/linux/capture-tools.html, branch david_auto_doc_gen_rebased. You can also view it on LunarXchange under Linux version 1.2.999.0.

Reply:
GFXR is sometimes released on its own.  Will there be a way to generate the HTML for a standalone GFXR release?

Reply:
To generate documentation, you would need to run update_docs.sh in Vulkan-SDK-Packaging.  Running the script is easy, but you do have to have several Vulkan repos on your system and they must be already built. But it's doable.

Reply:
I'm not sure I can accept the deletion of all this documentation in this PR if the PR brings a requirement that to regenerate the docs I have to have the other repos checked out.  If there's a way for me to run a standalone script in gfxr repo without the Vulkan-SDK-Packaging stuff then that would probably work.